sunpy/map/tests/test_mapbase.py

```diff
@@ -329,58 +329,57 @@ def calc_new_matrix(angle):
     return np.matrix([[c, -s], [s, c]])
 
 
-def test_rotate():
-    aia_map = sunpy.map.Map(sunpy.AIA_171_IMAGE)
-    rotated_map_1 = aia_map.rotate(20)
+def test_rotate(aia171_test_map):
+    rotated_map_1 = aia171_test_map.rotate(20)
     rotated_map_2 = rotated_map_1.rotate(20)
-    assert rotated_map_2.center == rotated_map_1.center == aia_map.center
+    assert rotated_map_2.center == rotated_map_1.center == aia171_test_map.center
```

Comment:
Go ahead and remove this assertion.  It's no longer valid because `center` – the data coordinates of the center of the pixel array – doesn't have to be the center of rotation, and hence is not necessarily preserved through rotation.